src/summersplash2022/img/tidslinje.tsx

```diff
@@ -0,0 +1,147 @@
+const TimeLine = () => {
+  return (
```

Comment:
Denne er per nå ikke universelt utformet, det vil si at ingenting av tidslinjen er tilgjengelig for svaksynte med skjermlesere. Tror den enkleste måten å fikse det på er å bare gjøre en kjapp oppsummering i tekst format som en title her.